documentation/feature/laserpointer.md

```diff
@@ -6,21 +6,18 @@ group: feature
 parent: wiki
 ---
 
-## Overview
+## 1. Overview
 
-### Sub-feature 1
-Short description of sub-feature 1.
+### 1.1 Visible light laser
+ACE3 adds visible light laser, what's even better is that BI's laser pointers and supported modded ones also get this feature.
 
-### Sub-feature 2
-Short description of sub-feature 2.
+## 2. Usage
 
+### 2.1 Switching laser mode
+- For this feature you need to have a compatible side attachement.
+- Press <kbd> ctrl+L </kbd> (by default).
+- On the top right corner of the screen you should have a hint that the mode changed.
```

Comment:
How do you like:
"A hint indicating the mode switch will appear in the top right corner."